testData/preferences/mary.json

```diff
@@ -4,10 +4,10 @@
             "gpii-default": {
                 "name": "Default preferences",
                 "preferences": {
-                    "http://registry.gpii.net/common/onScreenKeyboardEnabled": true,
+                    "http://registry.gpii.net/common/onscreenKeyboard": true,
```

Comment:
@kaspermarkus , sure, I'll take care of the spreadsheet. `onScreenKeyboardEnabled` is/was not yet listed as an accepted common term.